spec/role_model_spec.rb

```diff
@@ -6,7 +6,8 @@
   let(:role_options) { {} }
 
   before(:each) do
-    model_class.instance_eval do
+    role_options = self.role_options # must be defined outside scope.
```

Comment:
@martinrehfeld this is needed because it must be defined outside scope so that it can be used inside the class_eval, instance_eval, instance_exec, doesn't matter.

If you remove the line, all specs will fail.